sentry-android-core/src/main/java/io/sentry/android/core/ActivityLifecycleIntegration.java

```diff
@@ -67,7 +66,7 @@
 
   private @Nullable ISpan appStartSpan;
   private final @NotNull WeakHashMap<Activity, ISpan> ttidSpanMap = new WeakHashMap<>();
-  private @NotNull Date lastPausedTime = DateUtils.getCurrentDateTime();
+  private @NotNull SentryDate lastPausedTime = AndroidDateUtils.getCurrentSentryDateTime();
```

Comment:
Not sure about consequences, maybe @romtsn or @marandaneto can confirm?

Reply:
@stefanosiano was one implementing it, he might have opinions. But I don't see any problems with moving this into `register` since it's called shortly after this integration is instantiated anyway.